quesma/model/query.go

```diff
@@ -80,7 +81,14 @@ type QueryRowsTransfomer interface {
 	Transform(ctx context.Context, rows []QueryResultRow) []QueryResultRow
 }
 
+const MainExecutionPlan = "main"
+const AlternativeExecutionPlan = "alternative"
+
 type ExecutionPlan struct {
+	Name string
+
+	IndexPattern string
```

Comment:
Add `*table.ClickHouse`

Reply:
This is an issue.  It leads to cycle dependency. `clickhouse` package imports `model`. It can be a bigger refactor. I don't know if it should be done in this PR.  @pdelewski has a similar issue. Let's examine our packaging policy before.